### ISSUE TITLE
Different shops

### ISSUE THREAD

**Issue by LibreHacker** [jdfossapps/shop-with-mom#2]
You can add support different shop, for creating date base for prices and compare is with graph.

I think need soft by food item for this.

**Comment by jdfossapps** [comment]
Could you elaborate more on your suggestion? Do you mean saving a store for each purchase and then compare 2 shopping lists? Or am I missing something?

**Comment by LibreHacker** [comment]
Yes, and having history with prices. Sometimes it's can changes.

Also maybe create simple server with saving base for working with more users, or sharing it QR.

Sales can marked visually.

**Comment by jdfossapps** [comment]
Got it. Let me see what I can do. Not shure about the server part ... unless there is a foss server alternative that we can use.

**Comment by mg6nq8oe** [comment]
If we talk about server. I would say we should store it on wikidata.org . Is then of course public.

**Comment by jdfossapps** [comment]
Let me look into wikidata.org

**Comment by NitroNils** [comment]
Pardon me if I'm hijacking this issue, but I agree that it would be great to be able to build up/read a database of prices in different shops — I've been dreaming of an app like this for a while.
Another feature I'd love to see would be to scrape offers from sites/pdf's and update the shops offers.
Or manually let people collaborate on keeping prices updated by quick photo of price sign (OCR) — but I see this is adding a bunch of complexity so I'd better start learning to program 😁

Thanks for this project!
